janusgraph-dist/src/assembly/cfilter/conf/janusgraph-cassandra-configurationgraph.properties

```diff
@@ -81,3 +81,4 @@ cache.db-cache-time = 180000
 # Data Type:  Double
 # Mutability: MASKABLE
 cache.db-cache-size = 0.5
+#JANUSGRAPHCFG{gremlin.graph=org.janusgraph.core.JanusGraphFactory}
```

Comment:
This setting should not be added for properties files designed for configured graph factory.